components/Onboarding/ConnectViaWallet/ConnectViaWalletSupportedWallets.ts

```diff
@@ -44,6 +67,36 @@ export const POPULAR_WALLETS = [
   },
 ];
 
+export const DEFAULT_SUPPORTED_CHAINS = [
```

Comment:
Can we move this into a utils or constants file?
This way we can import it easier without also needing everything else in the file, long term it also avoids circular dependencies